### PR TITLE
Lobby is complete!

### PR DESCRIPTION
**Pull request by gullskatten**
It is now possible to:

* Create a new lobby (code is generated).
* Join a lobby with a code.
* Leave a lobby (as an opponent, after joining).
* Get notified about opponent left (returns to "waiting for opponent to join").
* Start the game after opponent connects.
* Get notified about lobby owner cancelling the lobby.
* Get a "waiting for opponent to start game" on successful lobby code input.

### CODE REVIEW

**Review by olaplassen — APPROVED** [review]
10/10 meget sterkt!

File: core/src/main/kotlin/no/ntnu/beardblaster/screen/LoadingScreen.kt [hunk 1]
```diff
@@ -55,9 +59,10 @@ class LoadingScreen(
             Nls.i18nBundle = assets[I18NAsset.Default]
             createSkin(assets)
             addGameScreens()
-            when (UserAuth().isLoggedIn()) {
-                true -> game.setScreen<MenuScreen>()
-                false -> game.setScreen<LoginMenuScreen>()
```
Comment (einaru) [inline core/src/main/kotlin/no/ntnu/beardblaster/screen/LoadingScreen.kt]: Noen grunn til at denne er revertet tilbake til en `if-else`?

**Review by einaru — APPROVED** [review]
Litt av et stykke arbeid her altså! Bra jobba 👍🏻 💯

File: core/src/main/kotlin/no/ntnu/beardblaster/screen/LoadingScreen.kt (hunk 1 above)
Comment (gullskatten) [reply core/src/main/kotlin/no/ntnu/beardblaster/screen/LoadingScreen.kt]: haha, ingen grunn faktisk! 😂

**Pull request merged**